published/004-2016-Jan-25.md

```diff
@@ -0,0 +1,92 @@
+# IPFS Weekly #4
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [January 25th](//github.com/ipfs/pm/issues/84) sprint:
+
+## Updates
+
+* [**js-ipfs**](//github.com/ipfs/js-ipfs): [@diasdavid](//github.com/diasdavid) created js-ipfs-merkle-dag and js-ipfs-blocks, which will help IPFS deal with MerkleDAGs and are extensible enough to allow others to build their own MerkleDAG structures. Also, js-ipfs is now able to import a file into the MerkleDAG, an important milestone. An architecture graph has been spec'ed out [here](https://github.com/ipfs/js-ipfs#ipfs-core-implementation-architecture) to help newcomers understand how js-ipfs works. Finally, lots of new issues and low-hanging fruit have been tagged for [contributors to tackle](//github.com/ipfs/js-ipfs/issues).
+* [**idb-plus-blob-store**](https://github.com/dignifiedquire/idb-plus-blob-store): Node streams are not able to provide consistent guarantees for writes on the "end" event, so [@dignifiedquire](//github.com/dignifiedquire) found a way to make them do that so that we can use them in js-ipfs. This fixes some inconsistencies that substack/idb-blob-store has.
+* [**go-ipfs**](//github.com/ipfs/go-ipfs): Thanks to [@Kubuxu](//github.com/Kubuxu), dnslink is now working. This allows users to alias their main domain to the ipfs.io gateway (using a CNAME), and still be able to set dnslinks separately. Also, [@whyrusleeping](//github.com/whyrusleeping) has [an open PR](https://github.com/ipfs/go-ipfs/pull/2256) to pull libp2p out of go-ipfs and put it into a module on [gx](https://github.com/whyrusleeping/gx), the IPFS native package manager. This is part of a larger effort to whittle go-ipfs down into smaller, extensible modules.
```

Comment:
s/dnslink/dnslink in subdomain (`_dnslink.`)/

Or something like that. dnslinks itself were working it is now working also under dedicated subdomain.